### PR TITLE
Mw/dev 17 7

### PR DESCRIPTION
**Pull request by michaelwood**

### CODE REVIEW

**Comment by michaelwood** [comment]
[Status Page: Add feed listing to status page](https://trello.com/c/SJ3CFSM3/104-status-page-add-feed-listing-to-status-page)

File: routes/index.js [hunk 1]
```diff
@@ -28,13 +30,43 @@ router.get('/', async function(req, res, next) {
 
   } catch(err) {
     error = err;
+    console.warn(err);
   }
 
   res.render('index', {
     statusData: statusData,
+    statusUrl, statusUrl,
```
Comment (odscjames) [inline routes/index.js:38]: statusUrl: statusUrl ??

File: routes/index.js [hunk 2]
```diff
@@ -28,13 +30,43 @@ router.get('/', async function(req, res, next) {
...
+    publisherDetails = await res.json();
+
+  } catch(err) {
+    error = err;
+    console.warn(err);
+  }
+
+  res.render('publisher_detail', {
+    publisherDetails: publisherDetails,
+    publisherDetailsUrl, publisherDetailsUrl,
```
Comment (odscjames) [inline routes/index.js:65]: Same as last comment?

**Comment by odscjames** [comment]
I would raise whether it's better as a separate page than a AJAX load as I'm sure we'll be adding more data per publisher and better for SEO but I'll assume that was already discussed at a design stage

File: routes/index.js (hunk 1 above)
Comment (michaelwood) [reply routes/index.js:38]: Could you clarify what you're asking here. Are you wondering what `statusUrl` is used for? see https://github.com/openactive/conformance-status-page/pull/10/files#diff-0fee1be8218cc43a268be5a23c313f2fR46

```html
   <p><a class="btn btn-secondary" href="{{statusUrl}}">Download as JSON</a></p> 
```

**Comment by michaelwood** [comment]
> I would raise whether it's better as a separate page than a AJAX load as I'm sure we'll be adding more data per publisher and better for SEO but I'll assume that was already discussed at a design stage

Like you say that decision is for the design. Also SEO don't need static HTML pages, most modern crawlers use a full browser engine (like headless webkit based browser) to crawl the page.

File: routes/index.js (hunk 1 above)
Comment (odscjames) [reply routes/index.js:38]: Nope, I'm asking why the code is "statusUrl, statusUrl" and not "statusUrl: statusUrl" - comma not colon -  is this a JS thing I don't know?

**Comment by odscjames** [comment]
> Also SEO don't need static HTML pages, most modern crawlers use a full browser engine (like headless webkit based browser) to crawl the page.

True, but I know what a static page crawler does and the details of this is usually a secret. Also there is deep linking. 

But yes, this is not the place for this discussion as long as it was already discussed elsewhere which is all I was checking?

**Comment by odscjames** [comment]
We probably want a "loading" indicator on the AJAX call but I'm happy for that to be part of tidy up later and for that not to be a blocker.

File: routes/index.js (hunk 1 above)
Comment (michaelwood) [reply routes/index.js:38]: No that's a typo, thanks will fix

**Pull request merged**